README.md

```diff
@@ -14,13 +14,39 @@ make
 sudo make install
 ``` 
 
-The example `.go` files are located in the `client` and `server` directories. 
-
-Run the command below to build and install the go-wolfssl wrapper.
+Then run the command below to build and install the wrapper module.
 ```
 go get -u github.com/wolfssl/go-wolfssl 
 ```
 
+## Building with configure options
+
+If building wolfSSL with a non-empty configure, you'll have to run the `./generateOptions` executable to customize go-wolfssl to the same feature set as wolfSSL. This executable will generate an `options.go` file that will keep go-wolfssl and wolfSSL in sync. `generateOptions` should be run any time you change your wolfSSL configure options. 
+
+Example: To use go-wolfssl without ECC, first build wolfSSL as shown below:
+```
+./configure --disable-ecc
+make
+sudo make install
+``` 
+
+Then go to the go-wolfssl directory and if the path to your wolfSSL directory is `../wolfssl`, just run:
+```
+./generateOptions
```

Comment:
How is `generateOptions` created?  It looks like one large (1.8Mb) binary file to me.  We like to ship source code, so would ideally want to convert this to a script that people could see the source of, or put the source of generateOptions in the repo with instructions on how to build it.

Reply:
Very good point. It was a `.go` app and the source code clashed with the wolfSSL package. Switched the binary with a `.sh` file

Reply:
Thanks for reworking this as a script, much better!  One more suggestion for the script though.  It seems that it would be more standard and easier for users if `generateOptions.sh` simply accepted an argument which represented their wolfSSL path.  Having to create a separate file (path.txt) seems a little non-standard to me.

For example:

```
$ ./generateOptions.h /path/to/wolfssl/directory
```

In your script, you can use the first argument provided by the user through the `$1` variable.

Reply:
Done